File: Python/Mahasiswa.py
from Sivitas import Sivitas

class Mahasiswa(Sivitas): 
    __nim = ""
    __semester = ""

    def __init__(self):
        super(Mahasiswa, self).__init__()
        self.nim = ""
        self.semester =""

    def setNim(self, a):
        self.nim = a
    def setSemester(self, a):
        self.semester = a
    
    def getNim(self):
        return self.nim
    def getSemester(self):
        return self.semester
    
    

File: CPP/Mahasiswa.cpp
#include <iostream>
#include <string>
#include "Sivitas.cpp"

using namespace std;

class Mahasiswa : public Sivitas
{
private:
    string nim;
    int semester;

public:
    Mahasiswa()
    {
        setNik("");
        setNama("");
        setJk("");
        setAsalUniv("");
        setEmailEdu("");
        nim = "";
        semester = 0;
    };
    Mahasiswa(string a, string b, string c, string d, string e, string f, int g, ProgramStudi h)
    {
        setNik(a);
        setNama(b);
        setJk(c);
        setAsalUniv(d);
        setEmailEdu(e);
        nim = f;
        semester = g;
        setProdi(h);
    }

    void setNim(string a)
    {
        nim = a;
    }
    void setsemester(int a)
    {
        semester = a;
    }

    string getNim()
    {
        return nim;
    }
    int getsemester()
    {
        return semester;
    }

    ~Mahasiswa()
    {
    }
};
